app/views/articles/show.html.erb

```diff
@@ -1,4 +1,48 @@
-<div>
-  <h1 class="font-bold text-4xl">Articles#show</h1>
-  <p>Find me in app/views/articles/show.html.erb</p>
+<%= link_to articles_path, class: "w-9 h-9 rounded-full grid place-items-center mx-auto text-gray-500 hover:bg-neutral-200/50 bg-transparent border border-neutral-300 hover:border-neutral-300 " do %>
+  <%= lucide_icon("arrow-left", class: "h-5 w-5") %>
+<% end %>
+
+<div class="max-w-[720px] mx-auto text-center space-y-4 mt-10">
+  <%= tag.div class: "min-h-[420px] rounded-lg bg-gradient-to-b #{article_gradient(@article)}" %>
+  <div class="max-w-[520px] mx-auto space-y-4">
+    <div class="space-y-4">
+      <span class="text-gray-500 font-medium text-xs uppercase">
+        <%= @article.created_at.to_date.to_formatted_s(:long) %> — <%= article_reading_time(@article) %>
+      </span>
+      <h1 class="text-neutral-900 font-medium text-3xl"><%= @article.title %></h1>
+    </div>
+
+    <% if @article.author_name.present? %>
+      <div class="flex flex-col items-center gap-1">
+        <%= image_tag @article.author_image, class: "w-10 h-10 rounded-full bg-gray-500 mb-3", alt: "Author" %>
+        <p class="uppercase text-gray-500 text-xs">Author</p>
+        <p><%= @article.author_name %></p>
+      </div>
+    <% end %>
+  </div>
 </div>
+
+<div class="max-w-[520px] mt-4 mx-auto prose">
+  <%= simple_format @article.content %>
+</div>
+
+<div class="mt-10">
+  <p class="text-xs font-medium uppercase text-center mx-auto text-gray-500">Share</p>
+  <div class="flex flex-row gap-4 justify-center items-center mt-4">
+    <%= link_to "#", class: "w-9 h-9 rounded-full grid place-items-center text-gray-500 hover:bg-neutral-200/50 bg-transparent border border-neutral-300 hover:border-neutral-300 " do %>
+      <%= lucide_icon("link-2", class: "h-5 w-5") %>
+    <% end %>
+    <%= link_to "#", class: "w-9 h-9 rounded-full grid place-items-center text-gray-500 hover:bg-neutral-200/50 bg-transparent border border-neutral-300 hover:border-neutral-300 " do %>
+      <%= lucide_icon("twitter", class: "h-5 w-5") %>
+    <% end %>
+    <%= link_to "#", class: "w-9 h-9 rounded-full grid place-items-center text-gray-500 hover:bg-neutral-200/50 bg-transparent border border-neutral-300 hover:border-neutral-300 " do %>
+      <%= lucide_icon("linkedin", class: "h-5 w-5") %>
+    <% end %>
+  </div>
+</div>
+
+<div class="grid grid-cols-[repeat(auto-fit,_minmax(min(288px,_100%),_1fr))] gap-4 mt-20">
+  <%= render Article.first(3) %>
```

Comment:
Should randomly select 3 articles, making sure to not select the currently article.